src/domain/notifications/notifications.repository.v2.ts

```diff
@@ -111,6 +106,8 @@ export class NotificationsRepositoryV2 implements INotificationsRepositoryV2 {
     chainId: string;
     safeAddress: `0x${string}`;
   }): Promise<Array<NotificationType>> {
+    // TODO: Is this even possible without an account?
+    // We'd need to theoretically need to store the preferences locally first?
```

Comment:
We need to discuss this with the clients as preferences (having subscribed _without_ a signature) are initially device-Safe related.